### PR TITLE
{bio} [goolf-1.4.10] OpenMM-6.1 Python-2.7.5 (REVIEW)

### PR DESCRIPTION
**Pull request by pescobar**

### CODE REVIEW

File: easybuild/easyconfigs/o/OpenMM/OpenMM-6.1-goolf-1.4.10-Python-2.7.5.eb [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+
+separate_build_dir = True
+
+python = 'Python'
+pyver = '2.7.5'
+pyshortver = '.'.join(pyver.split('.')[:2])
+versionsuffix = '-%s-%s' % (python, pyver)
+
+dependencies = [
+    (python, pyver),
```
Comment (boegel) [inline easybuild/easyconfigs/o/OpenMM/OpenMM-6.1-goolf-1.4.10-Python-2.7.5.eb]: @pescobar please use `%(pyver)s`?

File: easybuild/easyconfigs/o/OpenMM/OpenMM-6.1-goolf-1.4.10-Python-2.7.5.eb [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+runtest = ' test'
+
+preinstallopts = ' export OPENMM_INCLUDE_PATH=%(installdir)s/include && '
+preinstallopts += ' export OPENMM_LIB_PATH=%(installdir)s/lib && '
+
+# required to install the python API
+installopts = ' && cd python && python setup.py build && python setup.py install --prefix=%(installdir)s'
+
+sanity_check_paths = {
+    'files': ["lib/libOpenMM.so", "lib/python%s/site-packages/simtk/openmm/openmm.py" % (pyshortver)],
```
Comment (boegel) [inline easybuild/easyconfigs/o/OpenMM/OpenMM-6.1-goolf-1.4.10-Python-2.7.5.eb]: please use `SHLIB_EXT` and `%(pyshortver)s`?

File: easybuild/easyconfigs/o/OpenMM/OpenMM-6.1-goolf-1.4.10-Python-2.7.5.eb [hunk 3]
```diff
@@ -0,0 +1,59 @@
...
+    'dirs': []
+}
+
+modextrapaths = {
+    'PYTHONPATH': 'lib/python%s/site-packages' % (pyshortver),
```
Comment (boegel) [inline easybuild/easyconfigs/o/OpenMM/OpenMM-6.1-goolf-1.4.10-Python-2.7.5.eb]: please use `%(pyshortver)s`

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 2 out of 2 (2 easyconfigs in this PR)
node2001.delcatty.os - Linux SL 6.7, Intel(R) Xeon(R) CPU E5-2670 0 @ 2.60GHz, Python 2.6.6
See https://gist.github.com/d01e7b1d160214387c480a518c4c1e95 for a full test report.

**Comment by boegel** [comment]
@pescobar please look into the small remarks so we can get this merged?

**Comment by pescobar** [comment]
@boegel remarks fixed. please retest?

**Comment by boegel** [comment]
lgtm

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 2 out of 2 (2 easyconfigs in this PR)
node2159.delcatty.os - Linux SL 6.7, Intel(R) Xeon(R) CPU E5-2670 0 @ 2.60GHz, Python 2.6.6
See https://gist.github.com/79c3fe2ebcad4e8ef70aff7c06065169 for a full test report.

**Comment by boegel** [comment]
Going in, thanks @pescobar!

**Pull request merged**